docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -0,0 +1,480 @@
+description: scopri come impostare e personalizzare i collegamenti di navigazione della barra laterale del tuo sito Starlight.
+sidebar:
+  badge: Nuovo
+---
+
+import FileTree from '../../../../components/file-tree.astro';
+import SidebarPreview from '../../../../components/sidebar-preview.astro';
+
+Una barra laterale ben organizzata è fondamentale per una buona documentazione poiché è uno dei modi principali in cui gli utenti navigheranno nel tuo sito. Starlight fornisce un set completo di opzioni per personalizzare il layout e il contenuto della barra laterale.
+
+## Barra laterale predefinita
+
+Per impostazione predefinita, Starlight genererà automaticamente una barra laterale in base alla struttura del file system della documentazione, utilizzando la proprietà `title` di ciascun file come voce della barra laterale.
+
+Ad esempio, data la seguente struttura di file:
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - guides/
+        - components.md
+        - i18n.md
+      - reference/
+        - configuration.md
+
+</FileTree>
+
+Verrà generata automaticamente la seguente barra laterale:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'guides',
+			items: [
+				{ label: 'Components', link: '/guides/components/' },
+				{ label: 'Internationalization (i18n)', link: '/guides/i18n/' },
+			],
+		},
+		{
+			label: 'reference',
+			items: [
+				{ label: 'Configuration Reference', link: '/reference/configuration/' },
+			],
+		},
+	]}
+/>
+
+Scopri di più sulle barre laterali generate automaticamente nella sezione [gruppi generati automaticamente](#gruppi-generati-automaticamente).
+
+## Aggiungi collegamenti e gruppi di collegamenti
+
+Per configurare i [link](#link) e i [gruppi di link](#gruppi) della barra laterale (all'interno di un'intestazione comprimibile), utilizza la proprietà [`starlight.sidebar`](/it/reference/configuration/#sidebar) in `astro.config.mjs`.
+
+Combinando collegamenti e gruppi, puoi creare un'ampia varietà di layout della barra laterale.
+
+### Collegamenti
+
+Aggiungi un collegamento a una pagina interna o esterna utilizzando un oggetto con le proprietà `label` e `link`.
+
+```js
+starlight({
+	sidebar: [
+		// Un collegamento alla guida CSS e stile.
+		{ label: 'CSS e stile', link: '/guides/css-and-tailwind/' },
+		// Un collegamento esterno al sito web di Astro.
+		{ label: 'Astro', link: 'https://astro.build/' },
+	],
+});
+```
+
+La configurazione sopra genera la seguente barra laterale:
+
+<SidebarPreview
+	config={[
+		{ label: 'CSS e stile', link: '/guides/css-and-tailwind/' },
+		{ label: 'Astro', link: 'https://astro.build/' },
+	]}
+/>
+
+### Gruppi
+
+Puoi aggiungere struttura alla barra laterale raggruppando i collegamenti correlati sotto un'intestazione comprimibile.
+I gruppi possono contenere sia collegamenti che altri sottogruppi.
+
+Aggiungi un gruppo utilizzando un oggetto con le proprietà `label` e `items`.
+Il `label` verrà utilizzato come intestazione del gruppo.
```

Comment:
```suggestion
`label` verrà utilizzato come intestazione del gruppo.
```

Articles for english words in Italian are subjective. If we can avoid them, we can create less bikeshedding.

Reply:
To me it sounds slightly better with the article in front... without it sounds like `label` itself is used as the heading, and the value it assumes.

Reply:
I know, but I've seen people using feminine or masculine articles for the same words. E.g. la repository vs il repository 

Unless we stick to something (written somewhere), we will end up with mixed articles based on the person that writes the translation

Reply:
Ok, agreed. And committed.